internal/app/handlers/handlers.go

```diff
@@ -0,0 +1,102 @@
+package handlers
+
+import (
+	"io"
+	"net/http"
+	"strings"
+
+	"github.com/go-chi/chi/v5"
+
+	cnfg "URLShorter/internal/app/handlers/config"
+	repo "URLShorter/internal/app/repository"
+	sht "URLShorter/internal/app/service"
+)
+
+func Serve(cnf cnfg.Config, sht sht.Short) error {
+	h := NewHandlers(sht, cnf)
+	r := newRouter(h)
```

Comment:
```suggestion
	r := chi.NewRouter()
```
локальная функция `newRouter` только добавляет бессмысленную обертку над данным вызовом.